repositories04.list

```diff
@@ -13,3 +13,5 @@ https://github.com/muon-spectroscopy-computational-project/muon-galaxy-tools
 https://github.com/lldelisle/tools-lldelisle
 https://github.com/peterjc/galaxy_blast
 https://github.com/MaterialsGalaxy/larch-tools
+https://github.com/qiime2/galaxy-tools
```

Comment:
This one creates its own containers isn't it? So we do not need to create containers for them?

Reply:
We do not but it would be nice to have it (if possible) there for the Metadata extractor